src/docs/uxd/getStarted/DOC.md

```diff
@@ -1,3 +1,136 @@
+You and your product manager should schedule a cross-functional kickoff/working session with UXD, Dev, and Product Management. Contact Kendra Marchant (UXD for the platform) and Ian Hands (Product Management for the platform).
+To start, you’ll need to make some decisions around your application presence on cloud.redhat.com, and some of the user experience and UI depends on how you would like to entitle the product, how you plan for customers to access it. The platform is a place for Red Hat to deliver SaaS to our customers in a common way. For a given app on the platform, there may or may not be integrations with the other tenants.
+
+Once you’ve done that, work with the UX team to kickoff your design process. While there’s no one team that owns or approves all the apps in Cloud Services, the UX team creates and helps maintain design standards for every app in the platform and for the platform itself.
+
+## What is Cloud Services?
+
+Cloud.redhat.com, known internally and in this document as ‘Cloud Services,’ is a platform that presents bundles (groups) of SaaS applications to customers. The applications (apps) run in the cloud, so customers don’t have to install on-prem software.
+Important: Red Hat does not yet market the platform itself, only the capabilities provided on it. The platform has no official name at this time.
+
+Competitive examples for cloud.redhat.com include Microsoft Azure, AWS, and Google Cloud Platform.
+Red Hat Customers purchase subscriptions that provide entitlement to specific app bundles. A marketing website/page offering a try-and-buy experience is in discussion.
+Examples of bundles:
+
+* Red Hat Insights: All RHEL subscriptions include entitlement to Red Hat Insights. Insights is seen as the gateway to most of the other offerings on the platform.
+* Cloud Management Services for Red Hat Enterprise Linux: customers who purchase ‘Smart Management’ (includes Satellite) are entitled to the bundle ‘Cloud Management Services for RHEL’ (CMSfRHEL).  CMSfRHEL is sometimes referred to as ‘the upsell' from Insights.
+
+What does this platform and its apps help customers do?
+
+Red Hat Insights capabilities include:
+
+* Rules
+* Remediations (platform capability also available to CMSfRHEL)
+* Inventory (platform capability also available to CMSfRHEL)
+
+Cloud Management Services for Red Hat Enterprise Linux capabilities include:
+
+* Vulnerability
+* Compliance
+* System Comparison
+* Remediations
+* Inventory
+
+Cluster Manager (Dev Preview) is not yet PatternFly4 standard. Do not use to inform designs.
+
+Hybrid Cloud (Tech Preview June 1, Sept GA) capabilities include:
+
+* Catalog
+* Cost Management
+
+Subscriptions (Sept GA?)
+Its capabilities are TBD
+
+Cloud Services and released apps in production as of May 6:
+Live production site -- https://cloud.redhat.com
+
+Integration/staging site -- https://ci.cloud.paas.upshift.redhat.com/
```

Comment:
This is the outdated version:

the new one is https://ci.cloud.paas.psi.redhat.com/